wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -213,13 +243,12 @@ class PlatformRepo private constructor(val walletApplication: WalletApplication)
             nameDocuments.map { getIdentityForName(it) }
         }
 
-        var profileById: Map<Identifier, Document> = mapOf()
-
-        try {
-            val profileDocuments = Profiles(platform).getList(userIds)
-            profileById = profileDocuments.associateBy({ it.ownerId }, { it })
-        } catch (e: StatusRuntimeException) {
-            // swallow; we don't want to stop this method if no usernames have profiles
+        var profileById: Map<Identifier, Document> = if (userIds.isNotEmpty()) {
+            val profileDocuments = profiles.getList(userIds)
+            profileDocuments.associateBy({ it.ownerId }, { it })
+        } else {
+            log.warn("search usernames: userIdList is empty, though nameDocuments has ${nameDocuments.size} items")
+            mapOf()
```

Comment:
There was an exception here also, if userIds was empty.  This is related to the change above, when a search results in no results.